### ISSUE TITLE
setPreviewEffect doesn't work

### ISSUE THREAD

**Issue by nasko700** [CameraKit/camerakit-android#491]
### Is this a bug report?

I think there is a bug with the photo version of this library - more specifically 'com.camerakit:camerakit:1.0.0-beta3.11'.

### Steps to Reproduce
Just use this function to create CameraKitView in Fragment:
protected CameraKitView createCamera(Context context){
        CameraKitView cameraKitView = new CameraKitView(context);
        cameraKitView.setAdjustViewBounds(true);
        cameraKitView.setPreviewEffect(CameraKit.PREVIEW_EFFECT_SEPIA); // <-- HERE
        cameraKitView.setFacing(CameraKit.FACING_BACK);
        cameraKitView.setFlash(CameraKit.FLASH_OFF);

        return cameraKitView;
}

My device for testing is LG Nexus 5, API 23, Android 6.0.1. The device supports this color effect (Tested with own camera project).

### Expected Behavior

I expect to see the preview color effect Sepia, but I can't see it. 

### Actual Behavior

It doesn't apply cameraKitView.setPreviewEffect(CameraKit.PREVIEW_EFFECT_SEPIA);
in other words parameters.setColorEffect(Camera.Parameters.EFFECT_SEPIA).
It's the same for all other color effects.

### Reproducible Demo

Link to demo project: https://ufile.io/qzg7k (link will work for 30 days from now)

Offtopic question: I also wanna ask if someone has an idea when will be the expected time to have the ARCore part of this nice lib. Thanks!
